src/redux/redux-store.ts

```diff
@@ -18,10 +18,14 @@ let reducers = combineReducers ({
   app: appReducer
 });
 
+type RootReducerType = typeof rootReducer;
+export type AppStateType = ReturnType<RootReducerType>
+
+// @ts-ignore
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
-const store = createStore(reducers, composeEnhancers(applyMiddleware(thunkMiddleware)));
+const store = createStore(rootReducer, composeEnhancers(applyMiddleware(thunkMiddleware)));
 
-// let store = createStore(reducers, applyMiddleware(thunkMiddleware));
+// let store = createStore(rootReducer, applyMiddleware(thunkMiddleware));
 
 // TODO: Удалить, но потом
 // window.store = store;
```

Comment:
remove!

Reply:
For educational purposes I will leave this code